pypeit/requirements.txt

```diff
@@ -14,6 +14,7 @@ IPython>=7.2.0
 ginga>=3.0
 numba>=0.39.0
 linetools==0.3.dev2147
-PySide2 >= 5.0
+PySide2>=5.0
+#shapely>=1.7
 #pytest>=6.0.0   # JXP -- Started failing Travis on 02 Oct 2020 [not sure why]
```

Comment:
Since I see this here, can you turn this one back on and see if 
the new GitHub `CI` works with it?  Thanks.

Reply:
Yeah, will do when I push next.

Reply:
Yep, Github CI passes with inclusion of pytest in the requirements!  I uncommented shapely too, just as a check.  I'll comment it out, but leave pytest as is.